File: FrontEnd/src/app/features/manifest/classes/view-models/manifest-vm.ts
import { ManifestPassengerVM } from './manifest-passenger-vm'
import { ManifestShipVM } from './manifest-ship-vm'

export interface ManifestVM {

    date: string
    destination: string
    ship: ManifestShipVM
    passengers: ManifestPassengerVM[]

}

File: API/Features/Manifest/ViewModels/ManifestVM.cs
using System.Collections.Generic;

namespace API.Features.Manifest {

    public class ManifestVM {

        public string Date { get; set; }
        public string Destination { get; set; }
        public IEnumerable<PassengerVM> Passengers { get; set; }
        public IEnumerable<PassengerVM> Crew { get; set; }

    }

}